tasks/add_new_prebid_partner.py

```diff
@@ -1,8 +1,12 @@
 #!/usr/bin/env python
-# -*- coding: utf-8 -*-
+# encoding=utf8
 
 import logging
 import sys
+
+reload(sys)
+sys.setdefaultencoding('utf8')
```

Comment:
Remove the encoding changes, which are outside the scope of the issue you're addressing.